README.md

```diff
@@ -74,6 +74,26 @@ To get a local copy up and running follow these simple example steps.
 ![Chris Siku's GitHub stats](https://github-readme-stats.vercel.app/api?username=Chrissiku&count_private=true&theme=dark&show_icons=true)
 
```

Comment:
- Please, kindly remove this extra added content following the microverse [readme template](https://github.com/microverseinc/readme-template) to be consistent. This part could be added to the overview of your GitHub account.